File: TossSample/TossSampleAos/app/src/main/java/com/skw/tosssampleaos/TossReq.kt
package com.skw.tosssampleaos

import com.google.gson.annotations.SerializedName

/**
 * 토스 본인인증 표준창 정보 요청 필드
 */
data class TossSignAuthReq(
    val requestType: String
)

/**
 * 토스 본인인증 결과 요청 필드
 */
data class TossSignAuthResultReq(
    val txId: String,
    val sessionKey: String
)

File: TossSample/TossSampleIos/TossSampleIos/TossReq.swift
//
//  TossReq.swift
//  TossSampleIos
//
//  Created by skw on 8/24/23.
//

import Foundation

/// 토스 본인인증 표준창 정보 요청 필드
struct TossSignAuthReq: Codable {
    var requestType: String = ""
}

struct TossSignAuthResultReq: Codable {
    var txId: String = ""
    var sessionKey: String = ""
}
